guide/14-deep-learning/use_mistral_llm_for_text_classification_and_entity_recognition.ipynb

```diff
@@ -0,0 +1,615 @@
+{
```

Comment:
<p>T</p><p>Below are the parameters to be passed into <code>TextClassifier</code>: </p><p><code>backbone</code>: To use mistral as the model backbone, use&nbsp;<em>backbone="mistral"</em>.</p><p><code>examples</code>: User defined examples to provide the mistral model, in python dictionary format:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>